admin/conf/routes

```diff
@@ -46,6 +46,9 @@ POST        /page/clear
 GET         /ajax/clear                                                                             controllers.cache.PageDecacheController.renderAjaxDecache()
 POST        /ajax/clear                                                                             controllers.cache.PageDecacheController.decacheAjax()
 
+# Interactive Pressing
+GET         /interactive-librarian/live-presser/*path                                               controllers.admin.InteractiveLibrarianController.liveContentsPress(path: String)
```

Comment:
I think you should choose a different verb for this endpoint (`POST` perhaps), since it causes a change of state.

Reply:
Sorry, I misunderstood how this feature is intended to be used.  Please ignore this comment.

Reply:
Oh yes. Thanks David. I meant to do that and forgot!